redactions/widget.js

```diff
@@ -186,7 +186,7 @@ function fetchRasterBandInfo(datasetId, bandName) {
  * @param {string} layerId Layer ID
  */
 function fetchLayer(datasetId, layerId) {
```

Comment:
Change this to `getLayer`, `fetchLayer` should be keeped.

getX => internal call in the app.
fetchX => external call to other services.